### PR TITLE
Added a new resource link in database/youtube/web3-metaverse.json

### LINKED ISSUE #2054: [Add] new youtube channel link for web3 technologies
### Site URL

https://www.youtube.com/@web3mantra

### Description of the above provided site

This YouTube channel has 114 educational videos based on web3 technologies like blockchain, solidity, hardhat, smart contracts, etc. It is the best resource for web3.

### Category

Youtube

### Sub Category

web3

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by AmanPathan**
## Fixes Issue
Closes #2054

## Changes proposed
Added a new resource link in web3-metaverse.json

### CODE REVIEW

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

File: database/youtube/web3-metaverse.json [hunk 1]
```diff
@@ -110,5 +110,13 @@
     "category": "youtube",
     "subcategory": "web3-metaverse",
     "language": "english"
+  },
+  {
+    "name": "web3Mantra",
+    "description": "This channel has 114 educational videos on technologies like Web3, Blockchain, Solidity and Hardhat, ect",
```
Comment (CBID2) [inline database/youtube/web3-metaverse.json:116]:
```suggestion
    "description": "This channel has 114 educational videos on technologies like Web3, Blockchain, Solidity, Hardhat etc",
```

**Review by CBID2 — APPROVED** [review]
Just a few typos @AmanPathan

**Review by CBID2 — APPROVED** [review]
Looks good to me! :) Your turn @rupali-codes

**Pull request merged**